src/sentry/conf/server.py

```diff
@@ -438,6 +438,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     'sentry.tasks.scheduler', 'sentry.tasks.signals', 'sentry.tasks.store', 'sentry.tasks.unmerge',
     'sentry.tasks.symcache_update', 'sentry.tasks.servicehooks',
     'sentry.tagstore.tasks', 'sentry.tasks.assemble', 'sentry.tasks.integrations',
+    'sentry.tasks.files',
```

Comment:
You have `files` here, and the file is named `file.py`, so this won't add up.

Reply:
man im not on my game today